modules/binding-infra-node-workloads-using-taints-tolerations.adoc

```diff
@@ -1,20 +1,48 @@
 // Module included in the following assemblies:
 //
 // * machine_management/creating-infrastructure-machinesets.adoc
+// * post_installation_configuration/cluster-tasks.adoc
 
 [id="binding-infra-node-workloads-using-taints-tolerations_{context}"]
 = Binding infrastructure node workloads using taints and tolerations
 
 If you have an infra node that has the `infra` and `worker` roles assigned, you must configure the node so that user workloads are not assigned to it.
 
+[IMPORTANT]
+====
+It is recommended to preserve the dual `infra,worker` label that is created for infra nodes and use taints and tolerations to manage nodes user workloads are scheduled on. If you remove the `worker` label from the node, you must create a custom pool to manage it; a node with a label other than `master` or `worker` is not recognized by the MCO without a custom pool. Maintaining the `worker` label allows the node to be managed by the default worker machine config pool, and the `infra` label communicates to the cluster that it does not count toward the total number of subscriptions.
```

Comment:
I just realized this is actually poorly phrased. `Maintaining the `worker` label allows the node to be managed by the default worker machine config pool` is ONLY correct if no custom pools that select the custom label exists. Any custom pool labelling overrides the worker label in terms of pool.

To break down what I mean:
1. `infra, worker` label + `infra` pool = node in infra pool
2. `infra` label + `infra` pool = node in infra pool
3. `infra,worker` label + NO `infra` pool = node in worker pool
4. `infra` label + NO `infra` pool = unmanaged node

We never want them to be in 4, so to be safe we are advertising 1/3

Reply:
@yuqi-zhang  I changed to: _Maintaining the `worker` label allows the node to be managed by the default worker machine config pool, if no custom pools that select the custom label exists._

A user would maintain a `infra,worker` label in order to avoid an unmanaged node in the case that the infra pool goes away for some reason?